stdlib/public/core/ReflectionMirror.swift

```diff
@@ -47,10 +47,12 @@ internal func getChild<T>(of value: T, type: Any.Type, index: Int) -> (label: St
 internal func _getQuickLookObject<T>(_: T) -> AnyObject?
 
 @_silgen_name("_swift_stdlib_NSObject_isKindOfClass")
-internal func _isImpl(_ object: AnyObject, kindOf: AnyObject) -> Bool
+internal func _isImpl(_ object: AnyObject, kindOf: UnsafePointer<CChar>) -> Bool
```

Comment:
@jrose-apple I'm assuming `silgen_name` entities are not automatically `usableFromInline`, is that so?

Reply:
They are not, but it is sufficient to prevent dead code stripping (finally). Since this is only called from the runtime, which is in the same binary, this should be okay.